spec/factories/case_closure_metadata.rb

```diff
@@ -420,4 +443,38 @@
 
   end
 
+  factory :offender_ico_complaint_approval_flag, class: CaseClosure::OffenderComplaintApprovalFlag::ICOApprovalFlag do
+    subtype { nil }
+
+    trait :first_approval do
+      name                        { 'Has this had Branston operations approval?' }
+      abbreviation                { 'first_approval' }
+      sequence_id                 { 1000 }
+    end
+
+    trait :second_approval do
+      name                        { 'Has this had deputy director information services approval?' }
+      abbreviation                { 'second_approval' }
+      sequence_id                 { 1010 }
+    end
+
+    trait :not_approval_required do
+      name                      { 'No approval needed' }
+      abbreviation              { 'not_approval_required' }
```

Comment:
This is such a tiny thing, sorry, but I wonder if either "`approval_not_required`" or "`no_approval_required`" might read better?